dev-app/src/screens/HomeScreen.tsx

```diff
@@ -148,6 +150,10 @@ export default function HomeScreen() {
     loadDiscSettings();
   }, []);
 
+  const validTimeoutMethod = () => {
```

Comment:
This is currently valid for just bluetooth and usb discovery. Just a heads up that Android will be adding timeout support for discovering internet readers in the 4.0.0 release. I'm not too sure about when iOS will add it but might be a good idea to leave in a TODO for when both platforms provide support.

Reply:
Thanks for the heads up.
To me It's okey to leave it as it is because we also need to pass the timeout info later on when internet discovering is ready and that also require code change and test via the dev-app.